config/clusters/nasa-veda/common.values.yaml

```diff
@@ -79,6 +82,13 @@ basehub:
           profile_options:
             image: &image_options
               display_name: Image
+              unlisted_choice:
+                enabled: True
+                display_name: "Custom image"
+                validation_regex: "^.+:.+$"
+                validation_message: "Must be an image location, matching ^.+:.+$"
```

Comment:
Do we expect users to know what we mean when we present them with regex like this, or should we provide something more verbose/a link out to further resources?

Reply:
Btw, is that regex not just `something:something` and even matches `:::` or ` : `, for example? 

If the allowed pattern is indeed that wide, we can probably describe it as `image:tag`?